src/components/MailCredentialsModal.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import Modal from 'react-modal'
+import { CardContainer, CardHeader, CardHeaderNav } from './Card'
+import { CancelButton, SaveButton } from './buttons'
+import { CardForm, Input, Label } from './Form'
+
+export default function MailCredentialsModal({
+  setMailCreds,
+  mailCreds,
+  saveCreds,
+  modalIsOpen,
+  closeModal,
+  StyledModal,
+}) {
+  console.log(mailCreds)
+
+  return (
+    <Modal
+      isOpen={modalIsOpen}
+      onRequestClose={closeModal}
+      shouldCloseOnOverlayClick={false}
+      style={StyledModal}
+    >
+      <CardContainer>
+        <CardHeader>
+          <div></div>
+          <CardHeaderNav>
+            <CancelButton onClick={closeModal} />
+            <SaveButton type="submit" form="cardForm" onClick={handleSave} />
+          </CardHeaderNav>
+        </CardHeader>
+        <p>
+          Um die Benachrichtungsfunktion verwenden zu können, trage deine
+          E-Mail-Zugangsdaten hier ein:
+        </p>
+        <CardForm id="cardForm">
+          <Label htmlFor="host">
```

Comment:
You can put this code 
```
<small>
  <i>SMTP-Host</i>
</small>
```
into your Label component and transfer the text content with a prop.